client/src/components/Map/MAPMANUALTESTING.md

```diff
@@ -0,0 +1,13 @@
+GIVEN I am a user
+    WHEN I navigate to the home
+    AND I see the map component
+    AND I see the geo-location alert
+    AND I click cancel
+    THEN I see the map has been redirected to Sydney Australia
+
+GIVEN I am a user
```

Comment:
I really like the details on manual testing. Gives me a good idea of what behaviors we should be able to see.